src/index.ts

```diff
@@ -167,6 +174,27 @@ import { isNotEmptyString } from "utils";
             }
             break;
 
+          case KAFKA_PUBLIC_RELEASE_TOPIC:
+            if (FEATURE_INDEX_FILE_ENABLED) {
+              const event = parseFilePublicReleaseEvent(
+                message.value.toString()
+              );
+              if (isNotEmptyString(event.id)) {
+                await programQueueProcessor.enqueueEvent({
```

Comment:
add a new file event to donor aggregator queue when the file release message is valid.